c2corg_ui/static/js/gpxupload.js

```diff
@@ -59,6 +60,38 @@ app.GpxUploadController = function($scope) {
 
 
 /**
+ *
+ * @param {Array.<ol.Feature>} features - invalid features that have to be checked
+ * @returns {Array.<ol.Feature>} features - this geometry is valid
+ * @private
+ */
+app.GpxUploadController.prototype.validatedFeatures_ = function(features) {
+  for (var i = 0; i < features.length; i++) {
+    var geom = /**@type{ol.geom.Geometry}*/ (features[i].getGeometry());
+
+    if (geom instanceof ol.geom.MultiLineString) {
+      var multiLineString = /**@type{ol.geom.MultiLineString}*/ (geom);
+      var lineStrings = multiLineString.getLineStrings();
+
+      for (var j = 0; j < lineStrings.length; j++) {
+        if (lineStrings[j].getCoordinates().length === 1) {
+          delete lineStrings[j];
+        }
+      }
+      lineStrings = lineStrings.filter(function(n) {
+        return n != undefined;
+      });
```

Comment:
Isn't it possible to merge those 2 actions by directly removing the linestring from the list if it has only one set of coordinates?